src/lib/tabs/tab-nav-bar/tab-nav-bar.scss

```diff
@@ -17,8 +18,14 @@
   text-decoration: none;  // Removes anchor underline styling
   position: relative;
   overflow: hidden;  // Keeps the ripple from extending outside the element bounds
+
+  [mat-stretch-tabs] & {
```

Comment:
Shouldn't this be the same css targeting as for regular tabs?

Regular tabs are:
`.mat-tab-group[mat-stretch-tabs] .mat-tab-label`

Tab nav is:
`.mat-tab-link[mat-stretch-tabs]`

Should it be?
`mat-tab-links .mat-tab-link[mat-stretch-tabs]`

Reply:
This selector will end up being transpiled to `[mat-stretch-tabs] .mat-tab-link`. The only difference is that I didn't prepend the `.mat-tab-links`, but I wanted to make the selector less specific.

Reply:
Sorry, apparently I was not fully awake when I read this selector.  I see it now.  Awesome.